src/AhRulesBot/AhRulesBot/Program.cs

```diff
@@ -27,7 +29,9 @@ public static void Main(string[] args)
                             .AddSerilogLogging()
                             .AddTelegramBotClient()
                             .AddAhRulesFile()
-                            .AddHostedService<Worker>();
+                            .AddChannel()
```

Comment:
this is too generic name imo
It doesn't say anything about why we need that channel, or what is "channel"

Reply:
Fixed AddChannel => AddMsgToRemoveChannel